### PR TITLE
Update Link References In reference/docs-conceptual

### LINKED ISSUE #9314: 🎃 2022: Update Link References in docs-conceptual root Markdown files
This issue's related to [Hacktoberfest 2022][01]. For more info, see the meta-issue below:

- MicrosoftDocs/PowerShell-Docs#9257

[01]: https://hacktoberfest.com/participation/

### Prerequisites

- [X] **Existing Issue:** Search the existing Hacktoberfest issues for this repository. If there's an issue that covers the quality area and articles you want to improve, pick a different quality area or article set. Don't file a new issue for the same work. Subscribe, react, or comment on that issue instead.
- [X] **Descriptive Title:** Write the title for this issue as a short synopsis. If possible, provide context. For example, "🎃 2022: Command Syntax in Foo" instead of "🎃 2022: Foo"

### Quality Areas

Link References

### Article List

Limiting the scope to the markdown files in the /reference/docs-conceptual root based on this discussion:
https://github.com/MicrosoftDocs/PowerShell-Docs/issues/9257#issuecomment-1276343704

Files to be addressed in this Issue:
- https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/docs-conceptual/discover-powershell.md
- https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/docs-conceptual/how-to-use-docs.md
- https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/docs-conceptual/overview.md
- https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/docs-conceptual/powershell-commands.md

### PR DESCRIPTION
**Pull request by chadmando**
# PR Summary

This changes updates the link references in the markdown files located in the root folder of reference/docs-conceptual/ .

- Resolves #9314

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: reference/docs-conceptual/discover-powershell.md [hunk 1]
```diff
@@ -365,5 +365,9 @@ powerful help system.
...
 ### Additional resources
 
-- [Get-Command](xref:Microsoft.PowerShell.Core.Get-Command)
-- [Get-Member](xref:Microsoft.PowerShell.Utility.Get-Member)
+- [Get-Command][01]
+- [Get-Member][02]
+
+<!-- link references -->
+[01]: (xref:Microsoft.PowerShell.Core.Get-Command)
+[02]: (xref:Microsoft.PowerShell.Utility.Get-Member)
```
Comment (sdwheeler) [inline reference/docs-conceptual/discover-powershell.md:373]: You need to remove the parentheses when creating the reference.
```suggestion
[01]: xref:Microsoft.PowerShell.Core.Get-Command
[02]: xref:Microsoft.PowerShell.Utility.Get-Member
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
Please remove the parentheses from the reference definitions.

**Comment by sdwheeler** [comment]
@chadmando You need to remove the parentheses from all link references in all files.

**Comment by chadmando** [comment]
> @microsoft-github-policy-service agree

File: reference/docs-conceptual/overview.md [hunk 2]
```diff
@@ -42,72 +42,78 @@ manage almost any technology you work with. For example:
...
+[12]: /powershell/module/microsoft.powershell.utility/convertfrom-csv
+[13]: /powershell/module/microsoft.powershell.utility/convertfrom-json
+[14]: /powershell/module/microsoft.powershell.utility/convertto-xml
+[15]: (/powershell/azure)
+[16]: (/powershell/windows/get-started)
+[17]: (/powershell/exchange/exchange-management-shell)
+[18]: (/sql/powershell/sql-server-powershell)
+[19]: (https://aws.amazon.com/powershell/)
+[20]: (https://core.vmware.com/vmware-powercli)
+[21]: (https://cloud.google.com/powershell/)
```
Comment (sdwheeler) [inline reference/docs-conceptual/overview.md:107]: Remove the parentheses

File: reference/docs-conceptual/powershell-commands.md [hunk 3]
```diff
@@ -24,16 +24,23 @@ the cmdlet performs its action.
...
 
-- [Cmdlet overview](developer/cmdlet/cmdlet-overview.md)
+- [Cmdlet overview][05]
+
+<!-- link references -->
+[01]: (learn/tutorials/01-discover-powershell.md)
+[02]: (/powershell/module/microsoft.powershell.core/about/about_functions_advanced)
+[03]: (/powershell/module/microsoft.powershell.core/about/about_functions_cmdletbindingattribute)
+[04]: (/powershell/module/microsoft.powershell.core/about/about_functions_advanced_methods)
+[05]: (developer/cmdlet/cmdlet-overview.md)
```
Comment (sdwheeler) [inline reference/docs-conceptual/powershell-commands.md:46]: Remove the parentheses

**Review by sdwheeler — CHANGES_REQUESTED** [review]

**Comment by chadmando** [comment]
@sdwheeler If I would have **read** the build status messages, I would have figured it out sooner 😄 .  This is exactly why I wanted to do a small scope first.  I appreciate your patience.

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**